Cannot find module 'https-proxy-agent'

**Describe the bug**
Today the cli is starting reporting an error in a sub-dependency, breaking the execution.

**To Reproduce**
Steps to reproduce the behavior:
1.  `npx @redocly/openapi-cli@1.0.0-beta.32 `lint path-to-root-file.yaml`

**Expected behavior**
A clear and concise description of what you expected to happen.

**Logs**
```
npx: installed 99 in 8.928s
Cannot find module 'https-proxy-agent'
Require stack:
- /home/jenkins/.npm/_npx/240/lib/node_modules/@redocly/openapi-cli/node_modules/@redocly/openapi-core/lib/redocly/registry-api.js
- /home/jenkins/.npm/_npx/240/lib/node_modules/@redocly/openapi-cli/node_modules/@redocly/openapi-core/lib/redocly/index.js
- /home/jenkins/.npm/_npx/240/lib/node_modules/@redocly/openapi-cli/node_modules/@redocly/openapi-core/lib/config/load.js
- /home/jenkins/.npm/_npx/240/lib/node_modules/@redocly/openapi-cli/node_modules/@redocly/openapi-core/lib/index.js
- /home/jenkins/.npm/_npx/240/lib/node_modules/@redocly/openapi-cli/lib/utils.js
- /home/jenkins/.npm/_npx/240/lib/node_modules/@redocly/openapi-cli/lib/assert-node-version.js
- /home/jenkins/.npm/_npx/240/lib/node_modules/@redocly/openapi-cli/lib/index.js
- /home/jenkins/.npm/_npx/240/lib/node_modules/@redocly/openapi-cli/bin/cli.js
```

**OpenAPI definition**
If applicable, add an OpenAPI definition and `.redocly.yaml` configuration file that helps reproduce the problem.
At a minimum, please state the specification version(s) you're using (e.g. 2.0, 3.0, 3.1).

**`openapi-cli` Version(s)**
1.0.0-beta.32

**`Node.js` Version(s)**
nodejs 14, nodejs 16

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -42,7 +42,6 @@
     "glob": "^7.1.6",
     "glob-promise": "^3.4.0",
     "handlebars": "^4.7.6",
-    "https-proxy-agent": "^5.0.0",
     "portfinder": "^1.0.26",
     "simple-websocket": "^9.0.0",
     "yargs": "17.0.1"

--- a/packages/cli/src/commands/push.ts
+++ b/packages/cli/src/commands/push.ts
@@ -23,7 +23,6 @@ import {
   dumpBundle,
 } from '../utils';
 import { promptClientToken } from './login';
-const HttpsProxyAgent = require('https-proxy-agent');
 
 export async function handlePush(argv: {
   entrypoint?: string;
@@ -248,15 +247,12 @@ function uploadFileToS3(url: string, filePathOrBuffer: string | Buffer) {
       : filePathOrBuffer.byteLength;
   let readStream =
     typeof filePathOrBuffer === 'string' ? fs.createReadStream(filePathOrBuffer) : filePathOrBuffer;
-  const proxy = process.env.REDOCLY_PROXY;
-  const agent = proxy ? new HttpsProxyAgent(proxy) : undefined
 
   return fetch(url, {
     method: 'PUT',
     headers: {
       'Content-Length': fileSizeInBytes.toString(),
     },
-    agent,
     body: readStream,
   });
 }

--- a/packages/core/src/redocly/registry-api.ts
+++ b/packages/core/src/redocly/registry-api.ts
@@ -3,7 +3,6 @@ import { RegistryApiTypes } from './registry-api-types';
 import { AccessTokens, Region, DEFAULT_REGION, DOMAINS } from '../config/config';
 import { isNotEmptyObject } from '../utils';
 const version = require('../../package.json').version;
-const HttpsProxyAgent = require('https-proxy-agent');
 
 export class RegistryApi {
   constructor(private accessTokens: AccessTokens, private region: Region) {}
@@ -24,11 +23,9 @@ export class RegistryApi {
   private async request(path = '', options: RequestInit = {}, region?: Region) {
     const headers = Object.assign({}, options.headers || {}, { 'x-redocly-cli-version': version });
     if (!headers.hasOwnProperty('authorization')) { throw new Error('Unauthorized'); }
-    const proxy = process.env.REDOCLY_PROXY;
-    const agent = proxy ? new HttpsProxyAgent(proxy) : undefined;
     const response = await fetch(
       `${this.getBaseUrl(region)}${path}`,
-      Object.assign({}, options, { headers, agent }),
+      Object.assign({}, options, { headers }),
     );
     if (response.status === 401) { throw new Error('Unauthorized'); }
     if (response.status === 404) {

